contents/feedback/empty-status/simple/metadata.json

```
@@ -0,0 +1,15 @@
+{
+  "en": {
+    "title": "Simple",
+    "description": "It's a simple component for empty status."
+  },
+  "ja": {
+    "title": "シンプル",
+    "description": "空のステータスを表現したシンプルなコンポーネントです。"
+  },
+  "options": {
+    "container": {
+      "p": "md"
```

How about bringing it to the center?

```suggestion
      "p": "md",
      "centerContent": true
```

---

[73dc9f7](https://github.com/yamada-ui/yamada-components/pull/448/commits/73dc9f7c535a91c5fb6d80bc2ab13dedf763b9a8)